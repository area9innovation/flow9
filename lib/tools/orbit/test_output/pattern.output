QStandardPaths: wrong permissions on runtime directory /run/user/1000/, 0755 instead of 0700
Flow compiler (3rd generation)

Processing 'orbit' on http server
done in 0.13s

1
Result: 1
