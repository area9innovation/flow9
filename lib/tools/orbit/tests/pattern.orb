let a = Constructor(1);
a is (
	Constructor(n) => println(n)
)