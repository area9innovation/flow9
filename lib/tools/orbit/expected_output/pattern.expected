1
Result: 0
